utils/generic_utils.py

```diff
@@ -33,9 +33,13 @@ def load_config(config_path):
 
 
 def get_git_branch():
-    out = subprocess.check_output(["git", "branch"]).decode("utf8")
-    current = next(line for line in out.split("\n") if line.startswith("*"))
-    return current.replace("* ", "")
+    try:
+        out = subprocess.check_output(["git", "branch"]).decode("utf8")
```

Comment:
This is a workaround for the docker container, since putting the `.git` folder into the container really breaks the caching (and requires you to upload ~120MB every time you make a small change). So without a `.git` folder in the container, we need to catch the exception. If you still want to have the versioning, you have to keep the container around after the experiment, which is reasonable

Reply:
It makes the docker workflow unusable, since any small change of code will trigger a 120 MB update instead of a 1MB update that then needs to be uploaded